contracts/linear/src/validator_pool.rs

```diff
@@ -402,6 +402,16 @@ impl LiquidStakingContract {
         self.validator_pool.update_weight(&validator_id, weight);
     }
 
+    pub fn update_weights(&mut self, validator_ids: Vec<AccountId>, weights: Vec<u16>) {
```

Comment:
Is it necessary to change the type of weight from `u16` to `u32`?

Suppose we have 200 validators, and weight range from 5 to 50, it's likely total weights will overflow.